Library/ViewModels/SettingsViewModel.swift

```diff
@@ -576,6 +611,34 @@ SettingsViewModelOutputs {
   public var outputs: SettingsViewModelOutputs { return self }
 }
 
+private func dateFormatter(for dateString: String?, state: ExportDataEnvelope.State) -> String {
+  let dateFormatter = DateFormatter()
+  dateFormatter.dateFormat = "yyyy-MM-dd"
+  let newDateFormatter = DateFormatter()
+  newDateFormatter.dateFormat = "MMM d"
+
+  let timeFormatter = DateFormatter()
+  timeFormatter.dateFormat = "HH-mm-sZ"
+  let newTimeFormatter = DateFormatter()
+  newTimeFormatter.timeZone = TimeZone(secondsFromGMT: TimeZone.current.secondsFromGMT())
```

Comment:
time shown was previously off by an hour. I thought this was the way to fix that. 🤔